src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/VedtakService.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.tilleggsstonader.sak.vedtak
+
+import no.nav.tilleggsstonader.sak.behandling.domain.Saksbehandling
+import no.nav.tilleggsstonader.sak.fagsak.Stønadstype
+import org.springframework.data.annotation.Id
+import org.springframework.stereotype.Repository
+import org.springframework.stereotype.Service
+import java.util.UUID
+
+data class VedtakTilsynBarn(
+    @Id
+    val behandlingId: UUID,
+    val perioder: List<String>,
+    val beregningsresultat: List<String>,
+)
+
+@Repository
+class VedtakTilsynBarnRepository {
+    /**
+     * Erstatt disse metodene når det er et faktiskt repository
+     */
```

Comment:
Gjøre det lettere å søke opp kommentarer som er lagt inn med mål om å fikse noe senere
```suggestion
    /**
     * TODO: Erstatt disse metodene når det er et faktiskt repository
     */
```